robot/Cumulus/resources/NPSP.py

```diff
@@ -1669,4 +1669,22 @@ def query_and_store_records_to_delete(self,object_name,**kwargs):
             for i in records:
                 self.salesforce.store_session_record(object_name,i)
 
+    @capture_screenshot_on_error
+    def verify_table_contains_row(self,table_name,record,**kwargs):
+        """verifies that batch number format table contains a record with given name
+        and record field contains specified value. Example usage:
+        Verify Table Contains Row | Batches | MyBatch | Batch Number=MyBatch-01
+        """
+        for key,value in kwargs.items():
+            locator=npsp_lex_locators['datatable'].format(table_name,record,key)
+            actual=self.selenium.get_text(locator)
+            print(f'actual value is {actual}')
+            if actual==value:
+                print(f'Table contains {record} with expected {key}={value}')
+            elif value=='None' and actual=='':
```

Comment:
Curious what would be the scenario where value='None' and actual=' '  ?
for empty string validation you can use if not <my_string>

Reply:
In the test case when I am passing key value pairs I have to pass like Batch Number=abc1 but if I want to validate that field doesn't contain a value I am using Batch Number=None then when I see the expected value as None the expectation is to see the actual value as empty string. I should see empty string only when I pass None as value